core/tm/routes/profesional.ts

```diff
@@ -218,35 +218,24 @@ router.get('/profesionales/guia', async (req, res, next) => {
 });
 
 router.get('/profesionales/matching', async (req, res, next) => {
-    const opciones = {};
-
-    if (req.query.documento) {
-        opciones['documento'] = req.query.documento;
-    }
-
-    if (Object.keys(opciones).length !== 0) {
-        opciones['profesionalMatriculado'] = true;
-        const profEncontrados: any = await Profesional.find(opciones);
-        const arrayProf = [];
-        let resultado;
-        if (profEncontrados) {
-            profEncontrados.forEach(element => {
-                resultado = {
-                    id: element.id,
-                    nombre: element.nombre,
-                    sexo: element.sexo,
-                    apellido: element.apellido,
-                    documento: element.documento,
-                    fechaNacimiento: element.fechaNacimiento
-                };
-                arrayProf.push(resultado);
-            });
-
-        }
-        res.json(arrayProf);
-    } else {
-        res.json();
+    if (!req.query.documento || !req.query.sexo) {
+        throw 'Error matching requiere documento y sexo';
```

Comment:
Ojo acá q se está lanzando una excepción sin handlear y no se está dando respuesta al request
```suggestion
        return next('Error matching requiere documento y sexo');
```

Reply:
Corregidooo